File: CORS/tests/submissions/Microsoft/SetCookie.php
<?php
//Prevent Caching
header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
header("Last-Modified: " . gmdate("D, d M Y H:i:s") . " GMT");
header("Cache-Control: no-store, no-cache, must-revalidate");
header("Cache-Control: post-check=0, pre-check=0", false);
header("Pragma: no-cache");

//Set Cookie
header("Access-Control-Allow-Origin: *");
header("Set-Cookie: cookieName=cookieValue");

print("'Set-Cookie: cookieName=cookieValue' Sent in HTTP Response Header");
?>

File: worklets/resources/set-cookie.py
def main(request, response):
    name = request.GET.first(b"name")
    value = request.GET.first(b"value")
    source_origin = request.headers.get(b"Origin", None)

    response_headers = [(b"Set-Cookie", name + b"=" + value),
                        (b"Access-Control-Allow-Credentials", b"true")]

    if source_origin:
        response_headers.append((b"Access-Control-Allow-Origin", source_origin))

    return (200, response_headers, u"")
